src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let totalValue = days * 40;
+
+  if (days >= 3 && days < 7) {
+    totalValue = days * 40 - 20;
+  } else if (days >= 7) {
+    totalValue = days * 40 - 50;
+  }
+
+  return totalValue;
```

Comment:
The function name 'calculateRentalCost' is descriptive and clear, which is good. However, the implementation uses hardcoded values for the daily cost, and the discount values, which should be replaced with named constants to improve code readability and maintainability.